### PR TITLE
Add all data into blazegraph

### PR DESCRIPTION
**Pull request by Shalsh23**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -517,7 +521,41 @@ $(BUILD_DIR)/profiles.ttl: $(BUILD_DIR)/evolutionary-profiles.ttl $(BUILD_DIR)/g
...
+ 	$(BLAZEGRAPH-RUNNER) load --informat=turtle --journal=$(DB_FILE) --properties=$(BLAZEGRAPH_PROPERTIES) --graph="http://kb.phenoscape.org/taxa/ics/" $(BUILD_DIR)/corpus-ics-taxa.ttl \
+ 	$(BLAZEGRAPH-RUNNER) load --informat=turtle --journal=$(DB_FILE) --properties=$(BLAZEGRAPH_PROPERTIES) --graph="http://kb.phenoscape.org/taxa/expect-scores/" $(BUILD_DIR)/taxa-expect-scores.ttl \
+ 	$(BLAZEGRAPH-RUNNER) load --informat=turtle --journal=$(DB_FILE) --properties=$(BLAZEGRAPH_PROPERTIES) --graph="http://kb.phenoscape.org/taxa/pairwise-sim/" $(BUILD_DIR)/taxa-pairwise-sim.ttl \
+ 	$(BLAZEGRAPH-RUNNER) load --informat=turtle --journal=$(DB_FILE) --properties=$(BLAZEGRAPH_PROPERTIES) --graph="http://kb.phenoscape.org/gene/ics/" $(BUILD_DIR)/corpus-ics-genes.ttl \
+ 	$(BLAZEGRAPH-RUNNER) load --informat=turtle --journal=$(DB_FILE) --properties=$(BLAZEGRAPH_PROPERTIES) --graph="http://kb.phenoscape.org/gene/expect-scores/" $(BUILD_DIR)/gene-expect-scores.ttl \
+ 	$(BLAZEGRAPH-RUNNER) load --informat=turtle --journal=$(DB_FILE) --properties=$(BLAZEGRAPH_PROPERTIES) --graph="http://kb.phenoscape.org/gene/pairwise-sim/" $(BUILD_DIR)/gene-pairwise-sim.ttl
```
Comment (balhoff) [inline Makefile]: These aren't the graph IRIs we discussed; the graphs we use are:
- `http://kb.phenoscape.org/`
- `http://kb.phenoscape.org/closure`
- `http://kb.phenoscape.org/sim/taxa`
- `http://kb.phenoscape.org/sim/genes`

File: Makefile [hunk 2]
```diff
@@ -517,7 +521,41 @@ $(BUILD_DIR)/profiles.ttl: $(BUILD_DIR)/evolutionary-profiles.ttl $(BUILD_DIR)/g
...
+# Artifacts:
+# 1. Phenoscape KB
+# 2. Closures = subclass + instance)
+# 3. taxa-sim = taxa-ics + taxa-expect-scores + taxa-pairwise-sim
+# 4. gene-sim = gene-ics + gene-expect-scores + gene-pairwise-sim
+
+$(BUILD_DIR)/blazegraph-loaded-all.jnl: $(BUILD_DIR)/phenoscape-kb.ttl \
+										$(BUILD_DIR)/subclass-closure.ttl $(BUILD_DIR)/instance-closure.ttl \
+										$(BUILD_DIR)/corpus-ics-taxa.ttl $(BUILD_DIR)/taxa-expect-scores.ttl $(BUILD_DIR)/taxa-pairwise-sim.ttl \
+										$(BUILD_DIR)/corpus-ics-genes.ttl $(BUILD_DIR)/gene-expect-scores.ttl $(BUILD_DIR)/gene-pairwise-sim.ttl
```
Comment (balhoff) [inline Makefile]: I would `rm` any existing database file before starting. Otherwise you could mix old and new content.

**Comment by balhoff** [comment]
Ready to review again?

File: slurm-script/kb_init.sh [hunk 3]
```diff
@@ -7,6 +7,8 @@
 #SBATCH --constraint=broadwell
 set -e # Abort if any command fails
 
-export JAVA_OPTS="-Xmx70G"
+export JAVA_OPTS="-Xmx120G"
+source /home/spshriva/softwares/python-virtual-environments/env3/bin/activate
```
Comment (balhoff) [inline slurm-script/kb_init.sh]: We can't have paths like this in the script. It won't work for anyone else.

File: init.sh [hunk 4]
```diff
@@ -7,4 +7,5 @@
 ##error-<id> - file containing error log
 ##slurm-<id>.out - contains output log
 
+source ./env.sh
```
Comment (balhoff) [inline init.sh]: This file doesn't exist on this branch.

**Comment by Shalsh23** [comment]
Fixes #126

**Comment by balhoff** [comment]
I made a couple small makefile edits.

**Comment by balhoff** [comment]
@Shalsh23 can you add any pending changes you might have, and then merge this branch? It's been open too long and things are getting confusing. :-) I think we should merge it and then address the remaining issues with some new PRs. Thanks!

**Pull request merged**